README.md

```diff
@@ -1,23 +1,22 @@
-# <img src="https://raw.githubusercontent.com/marticliment/WingetUI/main/src/wingetui/Assets/Images/icon.png" height="40">UniGetUI (formerly WingetUI)
+# <img src="https://raw.githubusercontent.com/marticliment/WingetUI/main/media/icon.png" height="40">UniGetUI
```

Comment:
Like this one

Reply:
> Yes, in theory they should be all ok

Change paths from marticliment/WingetUI to marticliment/UniGetUI

Reply:
Not tey, I have not renamed the repo yet, and don't worry about this, since github will redirect WingetUI to UniGetUI